### DISCUSSION TITLE
Using cert-manager generated certificates for TLS

### DISCUSSION THREAD

**Discussion by mariusstaicu** [emqx/emqx-operator#431] [Q&A] [answered]
Hi,

Is there a way to configure TLS emqx broker v5 with certificates obtained  by cert-manager, let's say letsencrypt ?

**Comment by Rory-Z** [comment, marked as answer]
You can generate TLS secret by cert-manager, and use `extraVolumes` and `extraVolumeMounts` mount the secret to EMQX container, and update EMQX listeners config

**Reply by mariusstaicu** [reply]
I will try, however how can I specify the listeners config via emqx resource ? Is it via the boostrapConfig ?

**Reply by Rory-Z** [reply]
> Is it via the boostrapConfig ?
Yes, in v2alpha1 and EMQX 5.0, you can use `.spec.bootstrapConfig` to set EMQX config, and also you can use EMQX Dashboard to update config
